### PR TITLE
fix(lotdetails): L3-3327 fix input labels on non-mobile breakpoints, …

### PR DESCRIPTION
**Pull request by sgrund14**
…make accordion controllable

**Jira ticket**

[L3-3327](https://phillipsauctions.atlassian.net/browse/L3-3327)

**Summary**

Make labels for inputs 16px on larger breakpoints than mobile
Make accordion controllable

consumed by https://github.com/PhillipsAuctionHouse/phillips-public-remix/pull/444

**Change List (describe the changes made to the files)**

- Additions, Changes, and Removals

**Acceptance Test (how to verify the PR)**

- Add step by step instructions on how to verify the change

**Regression Test**

- (Optional) Add verification steps to make sure this PR doesn't break old functionality

**Evidence of testing**

- Post logs, screenshots, etc

**Things to look for during review**

- [ ] PR title should correctly describe the most significant type of commit. I.e. `feat(scope): ...` if a `minor` release should be triggered.
- [ ] All commit messages follow convention and are appropriate for the changes
- [ ] All references to `phillips` class prefix are using the prefix variable
- [ ] All major areas have a `data-testid` attribute.
- [ ] Document all props with jsdoc comments
- [ ] All strings should be translatable.
- [ ] Unit tests should be written and should have a coverage of 90% or higher in all areas.

New Components

- [ ] Are there any [accessibility considerations](https://www.w3.org/WAI/ARIA/apg/patterns/) that need to be taken into account and tested?
- [ ] Default story called "Playground" should be created for all new components
- [ ] Create a jsdoc comment that has an Overview section and a link to the Figma design for the component
- [ ] Export the component and its typescript type from the `index.ts` file
- [ ] Import the component scss file into the `componentStyles.scss` file.

[L3-3327]: https://phillipsauctions.atlassian.net/browse/L3-3327?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/Input/_input.scss [hunk 1]
```diff
@@ -23,6 +23,10 @@ $lg: #{$px}-input--lg;
     &--hidden {
       @include hidden;
     }
+
+    @media (min-width: $breakpoint-md) {
+      font-size: 1rem;
```
Comment (scottdickerson) [inline src/components/Input/_input.scss:28]: it seems to me that the input label (and string inside it) should be rendered with a `<Text variant={TextVariants.string2}`?
https://www.figma.com/design/hMu9IWH5N3KamJy8tLFdyV/EASEL-Compendium%3A-Tokens%2C-Components-%26-Patterns?node-id=7584-17569&m=dev

image

Comment (sgrund14) [reply src/components/Input/_input.scss:28]: will that handle the breakpoints?

Comment (sgrund14) [reply src/components/Input/_input.scss:28]: or I guess I just use the fresnel thing

File: src/components/Accordion/Accordion.tsx [hunk 2]
```diff
@@ -32,20 +40,28 @@ export interface AccordionProps extends React.HTMLAttributes<HTMLDivElement> {
...
+    const variantProps = getAccordionVariantProps(props.variant);
+
+    return (
+      <Accordion.Root
+        className={classnames(`${baseClassName}`, className)}
+        {...commonProps}
+        {...variantProps}
+        id={props?.id}
+        ref={ref}
+        type="multiple"
```
Comment (scottdickerson) [inline src/components/Accordion/Accordion.tsx:55]: few problems here, `variantProps` is returning `collapsible: true` which is blowing up the domconsole
```
Warning: Received `true` for a non-boolean attribute `collapsible`.

If you want to write it to the DOM, pass a string instead: collapsible="true" or collapsible={value.toString()}.
    at div
    at http://localhost:6006/node_modules/.cache/storybook/1c3385a5d25e538d10b518b310c74d3ca2690b6aaffeadccd74da79736171f86/sb-vite/deps/chunk-6OGP3IOB.js?v=e5fade98:276:13
    at http://localhost:6006/node_modules/.cache/storybook/1c3385a5d25e538d10b518b310c74d3ca2690b6aaffeadccd74da79736171f86/sb-vite/deps/chunk-6OGP3IOB.js?v=e5fade98:130:11
```

Also, we can't hardcode the type to `multiple`, we have to use the type returned by `{...variantProps}`

**Review by scottdickerson — APPROVED** [review]
LGTM

**Pull request merged**